packages/rlc-common/src/test/template.ts

```diff
@@ -168,20 +168,18 @@ export async function createRecorder(context: Context): Promise<Recorder> {
 `;
 
 export const sampleTestContent = `
-import { Recorder } from "@azure-tools/test-recorder";
-import { assert } from "chai";
-import { createRecorder } from "./utils/recordedClient{{#if isModularLibrary}}.js{{/if}}";
-import { Context } from "mocha";
+// import { createRecorder } from "./utils/recordedClient{{#if isEsm}}.js{{/if}}";
+import { assert, beforeEach, afterEach, it, describe } from "vitest";
```

Comment:
are we going to use vitest to replace mocha and test recorder now?

Reply:
Yes, that is the plan to remove karma, mocha and we are fixing the test recorder to use either chai or vitest.